tests/wdio/Data/Setup.js

```diff
@@ -0,0 +1,55 @@
+const Page = require('../vaah-webdriverio/Page');
+
+class Setup extends Page{
+    constructor() {
+        super();
+        this.params = {
+            group: {
+                count: null,
+                name: null,
+            }
+        }
+        this.element = {
+            install_button: '[data-testid="setup-install_vaahcms"]'
```

Comment:
@we-abhijeet-g001 if you use https://github.com/webreinvent/vaahcms-ready/blob/2.x/tests/wdio/vaah-webdriverio/Selector.js#L64 that method, you need to any write `setup-install_vaahcms`